myapp/app/models/task.rb

```diff
@@ -3,6 +3,10 @@
 class Task < ApplicationRecord
   DEFAULT_PRIORITY_VALUE = 1
   DEFAULT_STATUS_VALUE = 0
+  SORT_TYPE = {
+    'id_asc' => 'tasks.id ASC',
+    'id_desc' => 'tasks.id DESC'
```

Comment:
`tasks`は明示的に書かなくても問題ないと思うのですがいかがでしょうか？

```suggestion
    'id_asc' => 'id ASC',
    'id_desc' => 'id DESC'
```

Reply:
確かに書かなくても問題ないので削除致します！